### PR TITLE
OSDOCS#8714: OSUS 5.0.3 release image signature changes

### PR DESCRIPTION
**Pull request by skopacz1**
[OSDOCS-8174](https://issues.redhat.com/browse/OSDOCS-8174)

Version(s): 4.16+

This PR adds changes enabled/introduced with the release of the OSUS Operator 5.0.3. The changes include the ability to configure the CVO to retrieve release image signatures from  an OSUS endpoint, preventing the need to manually apply the signatures with each new release image payload a user mirrors.

QE review:
- [ ] QE has approved this change.

Previews:

- [Configuring your cluster to use the resources generated by oc-mirror](https://74995--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#oc-mirror-updating-cluster-manifests_installing-mirroring-disconnected)
- [Creating the OpenShift Update Service graph data container image](https://74995--ocpdocs-pr.netlify.app/openshift-enterprise/latest/updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.html#update-service-graph-data_updating-restricted-network-cluster-osus)
- [Configuring the Cluster Version Operator (CVO) to retrieve release image signatures](https://74995--ocpdocs-pr.netlify.app/openshift-enterprise/latest/updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.html#configure-cvo-retrieve-image-signatures_updating-restricted-network-cluster-osus)
- [Next steps](https://74995--ocpdocs-pr.netlify.app/openshift-enterprise/latest/updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.html#next-steps_updating-restricted-network-cluster-osus)

### CODE REVIEW

File: modules/configure-cvo-retrieve-image-signatures.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+// * updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configure-cvo-retrieve-image-signatures_{context}"]
+= Configuring the Cluster Version Operator (CVO) to retrieve release image signatures
+
+If you are using the oc-mirror plugin 4.15 or later and the OSUS Operator 5.0.3 or later to update your cluster in a disconnected environment, you can configure the Cluster Version Operator (CVO) to retrieve release image signatures from an endpoint on the local OSUS Operator.
+This configuration is done by adding the URI of OSUS Operator's signature serving endpoint to the CVO's list of signature stores.
```
Comment (skopacz1) [inline modules/configure-cvo-retrieve-image-signatures.adoc:9]: I added this line based on the draft doc, but let me know if this is too much technical detail for the end user.

File: modules/update-service-graph-data.adoc [hunk 2]
```diff
@@ -5,13 +5,13 @@
 [id="update-service-graph-data_{context}"]
 = Creating the OpenShift Update Service graph data container image
 
-The OpenShift Update Service requires a graph data container image, from which the OpenShift Update Service retrieves information about channel membership and blocked update edges. Graph data is typically fetched directly from the update graph data repository. In environments where an internet connection is unavailable, loading this information from an init container is another way to make the graph data available to the OpenShift Update Service. The role of the init container is to provide a local copy of the graph data, and during pod initialization, the init container copies the data to a volume that is accessible by the service.
-
 [NOTE]
 ====
 The oc-mirror OpenShift CLI (`oc`) plugin creates this graph data container image in addition to mirroring release images. If you used the oc-mirror plugin to mirror your release images, you can skip this procedure.
 ====
```
Comment (skopacz1) [inline modules/update-service-graph-data.adoc:11]: I basically shifted this note about graph data images being unnecessary with oc-mirror higher up to the top of the section. 

But if anyone thinks this entire module should be deleted so that our "updating with OSUS" page only documents the oc-mirror + OSUS workflow, let me know.

File: updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc [hunk 3]
```diff
@@ -121,17 +121,20 @@ include::modules/update-service-configure-cvo.adoc[leveloffset=+3]
...
+include::modules/configure-cvo-retrieve-image-signatures.adoc[leveloffset=+1]
+
 [id="next-steps_updating-restricted-network-cluster-osus"]
 == Next steps
 
 Before updating your cluster, confirm that the following conditions are met:
 
 * The Cluster Version Operator (CVO) is configured to use your locally-installed OpenShift Update Service application.
-* The release image signature config map for the new release is applied to your cluster.
+* Your cluster is able to retrieve release image signatures for the update target release images.
```
Comment (skopacz1) [inline updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc:133]: Rather than adding an additional statement about how oc mirror 4.15+ and OSUS 5.0.3+ makes this step unnecessary, I just broadened this condition to state that the cluster can get access to signatures somehow. 

If it's worth specifying the two different ways that can happen depending on your workflow, I can add that in level two bullets underneath this one.

File: modules/configure-cvo-retrieve-image-signatures.adoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+This is a one-time configuration that prevents the need to apply release image signatures each time you mirror a new set of release images.
```
Comment (wking) [inline modules/configure-cvo-retrieve-image-signatures.adoc:11]: Maybe "prevents" -> "avoids"?  Not entirely sure why I feel that way; maybe I expect "prevents" for bad things and "avoids" for annoying things, and per-update signature application is annoying, but not bad?  Feel free to ignore if this makes no sense :)

File: modules/configure-cvo-retrieve-image-signatures.adoc (hunk 1 above)
Comment (wking) [reply modules/configure-cvo-retrieve-image-signatures.adoc:9]: I think this line is helpful, but it might be long in the same paragraph as the `This is a one-time configuration...` qualifier?

Comment (skopacz1) [reply modules/configure-cvo-retrieve-image-signatures.adoc:9]: That sounds good, I can move it to the other paragraph for now.

File: modules/configure-cvo-retrieve-image-signatures.adoc (hunk 4 above)
Comment (skopacz1) [reply modules/configure-cvo-retrieve-image-signatures.adoc:11]: I'm not super opinionated so "avoids" would be fine by me! The logic makes sense to me, personally

File: modules/configure-cvo-retrieve-image-signatures.adoc [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+.Procedure
+
+* Patch the cluster version resource by running the following command:
++
+[source,terminal]
+----
+$ oc patch clusterversion version --type=merge -p '{"spec":{"signatureStores":[{"url": "<signature_serving_url>"}]}}'
+----
+// Is the URL from the OSUS Operator going to be obvious to the user/easy for them to find? Or should we either give some hint or an example somewhere in this procedure?
```
[the remark this answers is not in the document]
Comment (skopacz1) [reply modules/configure-cvo-retrieve-image-signatures.adoc:29]: Is there a command we can offer to users to retrieve the URL from this property, or should we just mention the property in a note and leave it at that? The note could be worded like the following:
```suggestion
+
[NOTE]
====
You can find the signature serving URL of the OSUS Operator by searching for the `MetadataURI` property of `UpdateServiceStatus`.
====
```

Or whatever the technically accurate wording would be.

[the remark this answers is not in the document]
Comment (PratikMahajan) [reply modules/configure-cvo-retrieve-image-signatures.adoc:29]: or we can have something like this 
```shell
while sleep 1; do SIGNATURES_URI=“$(oc -n “${NAMESPACE}” get -o jsonpath=‘{.status.metadataURI}/api/upgrades_info/signatures/{“\n”}’ updateservice “${NAME}“)”; SCHEME=“${SIGNATURES_URI%%:*}“; if test “${SCHEME}” = http -o “${SCHEME}” = https; then break; fi; done
```

similar to https://docs.openshift.com/container-platform/4.15/updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.html#update-service-create-service-cli_updating-restricted-network-cluster-osus

[the remark this answers is not in the document]
Comment (skopacz1) [reply modules/configure-cvo-retrieve-image-signatures.adoc:29]: @PratikMahajan does that command print out the URI, or does it set the URI to `SIGNATURES_URI` which you'd then use in place of <signature_serving_url> in the actual command, like this?
`$ oc patch clusterversion version --type=merge -p '{"spec":{"signatureStores":[{"url": "${SIGNATURES_URI}"}]}}'`

File: modules/configure-cvo-retrieve-image-signatures.adoc [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+This configuration is done by adding the URI of OSUS Operator's signature serving endpoint to the CVO's list of signature stores through the `signatureStores` property.
+This is a one-time configuration that avoids the need to apply release image signatures each time you mirror a new set of release images.
+
+:FeatureName: The `signatureStores` property
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have installed the OSUS Operator 5.0.3 or later.
```
Comment (petr-muller) [inline modules/configure-cvo-retrieve-image-signatures.adoc:18]: I'd rephrase this as "You have deployed OSUS using OSUS Operator 5.0.3"

File: modules/configure-cvo-retrieve-image-signatures.adoc [hunk 7]
```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+// * updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configure-cvo-retrieve-image-signatures_{context}"]
+= Configuring the Cluster Version Operator (CVO) to retrieve release image signatures
+
+If you are using the oc-mirror plugin 4.15 or later and the OSUS Operator 5.0.3 or later to update your cluster in a disconnected environment, you can configure the Cluster Version Operator (CVO) to retrieve release image signatures from an endpoint on the local OpenShift Update Service (OSUS).
```
Comment (petr-muller) [inline modules/configure-cvo-retrieve-image-signatures.adoc:8]: > from an endpoint on the local OpenShift Update Service (OSUS).

I'm not sure about the "local". Our recommended topology is to have a single OSUS deployed somewhere (on one OCP cluster) and and let more (many!) OCP clusters consume its services. In technical context I'd read "local" as "running in the same cluster" which is not necessary and I'm worried that it reinforces the anti-pattern that you should run one OSUS inside every cluster.

I guess "local" can also mean "local to your organization; not RH-operated" but I'd still avoid call that "local" because of the possible confusion.

Comment (skopacz1) [reply modules/configure-cvo-retrieve-image-signatures.adoc:8]: If you look at the [rest of the page](https://docs.openshift.com/container-platform/4.15/updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.html) that this content is being added to and search for "local", you'll find many instances where we use this "local OSUS" wording. 

I don't disagree with implementing this change generally and in this PR, but I think that if we do try to shy away from this wording, we'd want to make a broader sweep throughout this page to make the change effective. 

Would it also be better to find another word to convey "not the OSUS hosted by RH"? I guess without it, the context of the page is probably enough to inform users, but maybe clarity wherever possible might be a safer bet.

What about this wording?
```suggestion
If you are using the oc-mirror plugin 4.15 or later and the OSUS Operator 5.0.3 or later to update your cluster in a disconnected environment, you can configure the Cluster Version Operator (CVO) to retrieve release image signatures from an endpoint on the OpenShift Update Service (OSUS) in your environment.
```

Comment (petr-muller) [reply modules/configure-cvo-retrieve-image-signatures.adoc:8]: Good point about consistency with other instances. I haven't searched but I agree that if we use it in more places then doing it in a separate, broad PR is preferable over changing a single instance in this PR :+1:

"in your environment" sounds good to me

Comment (skopacz1) [reply modules/configure-cvo-retrieve-image-signatures.adoc:8]: Sounds good! I guess I'll keep "local" in this PR and then I can create another Jira issue to cover the broader change. That'll also give us all more time to discuss possible wording options in addition to "in your environment"

**Review by PratikMahajan — APPROVED** [review]
/lgtm

**Comment by skopacz1** [comment]
@JianLi-RH could you PTAL when you have a chance? Thanks!

File: modules/configure-cvo-retrieve-image-signatures.adoc (hunk 7 above)
Comment (skopacz1) [reply modules/configure-cvo-retrieve-image-signatures.adoc:8]: Since I'm starting the work to update the "local" wording before this PR is finished, I'll go ahead and update this one instance here.

File: modules/configure-cvo-retrieve-image-signatures.adoc [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+* You have deployed OSUS using the OSUS Operator 5.0.3 or later.
+* You are using the oc mirror plugin 4.15 or later to mirror release images.
+
+.Procedure
+
+. Retrieve the signature serving URI of OSUS by running the following command:
++
+[source,terminal]
+----
+$ while sleep 1; do SIGNATURES_URI=“$(oc -n “${NAMESPACE}” get -o jsonpath=‘{.status.metadataURI}/api/upgrades_info/signatures/{“\n”}’ updateservice “${NAME}“)”; SCHEME=“${SIGNATURES_URI%%:*}“; if test “${SCHEME}” = http -o “${SCHEME}” = https; then break; fi; done
```
Comment (JianLi-RH) [inline modules/configure-cvo-retrieve-image-signatures.adoc:27]: It seems `“` should be `"`, `‘` should be `'`?

Comment (JianLi-RH) [reply modules/configure-cvo-retrieve-image-signatures.adoc:27]: I ran below command on my cluster which has OSUS and updateservice installed, but stucked:
```
[jianl@localhost 415]$ NAME=sample
[jianl@localhost 415]$ NAMESPACE=openshift-update-service
[jianl@localhost 415]$ while sleep 1; do SIGNATURES_URI="$(oc -n "${NAMESPACE}" get -o jsonpath='{.status.metadataURI}/api/upgrades_info/signatures/{"\n"}' updateservice "${NAME}")"; SCHEME="${SIGNATURES_URI%%:*}"; if test "${SCHEME}" = http -o "${SCHEME}" = https; then break; fi; done


```

File: modules/configure-cvo-retrieve-image-signatures.adoc [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+----
+
+. Patch the cluster version resource by running the following command:
```
Comment (JianLi-RH) [inline modules/configure-cvo-retrieve-image-signatures.adoc:30]: Because above command stuck, I can't run the patch command. 
But I can get SIGNATURES_URI by running below command:
```
[jianl@localhost 415]$ SIGNATURES_URI="$(oc -n "${NAMESPACE}" get -o jsonpath='{.status.metadataURI}/api/upgrades_info/signatures/{"\n"}' updateservice "${NAME}")"
[jianl@localhost 415]$ echo $SIGNATURES_URI
/api/upgrades_info/signatures/
[jianl@localhost 415]$ 
```

File: modules/oc-mirror-updating-cluster-manifests.adoc [hunk 10]
```diff
@@ -36,7 +36,11 @@ $ oc apply -f ./oc-mirror-workspace/results-1639608409/release-signatures/
 +
 [NOTE]
 ====
-If you are mirroring Operators instead of clusters, you do not need to run `$ oc apply -f ./oc-mirror-workspace/results-1639608409/release-signatures/`. Running that command will return an error, as there are no release image signatures to apply.
+* If you are using the oc-mirror plugin 4.15 or later and the OSUS Operator 5.0.3 or later to update your cluster in a disconnected environment, you can skip this step.
+However, the Cluster Version Operator must be configured to retrieve release image signatures from the OpenShift Update Service (OSUS).
+For more information, see "Configuring the Cluster Version Operator (CVO) to retrieve release image signatures" in the "Additional resources" section.
+
+* If you are mirroring Operators instead of clusters, you can skip this step. Running the previous command will return an error, as there are no release image signatures to apply.
```
Comment (JianLi-RH) [inline modules/oc-mirror-updating-cluster-manifests.adoc:43]: `If you are mirroring Operators instead of clusters` maybe made confusing, I think `If you are mirroring Operators instead of release images` or `If you are mirroring Operators instead of payload images` will be better, thought?

File: updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc [hunk 11]
```diff
@@ -114,24 +114,27 @@ and must be no more than 63 characters`, try creating the Update Service with a
...
 +
 [NOTE]
 ====
-The release image signature config map allows the Cluster Version Operator (CVO) to ensure the integrity of release images by verifying that the actual image signatures match the expected signatures.
+Release image signatures allow the Cluster Version Operator (CVO) to ensure the integrity of release images by verifying that the actual image signatures match the expected signatures.
```
Comment (JianLi-RH) [inline updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc:137]: It's hard for me to understand this text :( Any one have any idea to make it easy to understand? cc @jiajliu @shellyyang1989

File: modules/configure-cvo-retrieve-image-signatures.adoc [hunk 12]
```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+// * updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configure-cvo-retrieve-image-signatures_{context}"]
+= Configuring the Cluster Version Operator (CVO) to retrieve release image signatures
+
+If you are using the oc-mirror plugin 4.15 or later and the OSUS Operator 5.0.3 or later to update your cluster in a disconnected environment, you can configure the Cluster Version Operator (CVO) to retrieve release image signatures from an endpoint on the OpenShift Update Service (OSUS) in your environment.
+
+This configuration is done by adding the URI of OSUS' signature serving endpoint to the CVO's list of signature stores through the `signatureStores` property.
```
[the remark this answers is not in the document]
Comment (skopacz1) [reply modules/configure-cvo-retrieve-image-signatures.adoc:10]: Sure, I'm changing this line to:

You can configure this by adding the URI of OSUS' signature serving endpoint to the CVO's list of signature stores through the `signatureStores` property.

File: modules/configure-cvo-retrieve-image-signatures.adoc (hunk 8 above)
Comment (skopacz1) [reply modules/configure-cvo-retrieve-image-signatures.adoc:27]: @JianLi-RH I can fix the quotation marks, sure. 

When you ran the command, did you directly copy and paste from this line, or did you change the quotation marks yourself before running the command?

File: modules/oc-mirror-updating-cluster-manifests.adoc (hunk 10 above)
Comment (skopacz1) [reply modules/oc-mirror-updating-cluster-manifests.adoc:43]: Good catch, thanks! I think that was probably meant to say "cluster Operators" or something similar to your suggestions.

I can change it to say "release images" since that seems to make the most sense here.

File: updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc (hunk 11 above)
Comment (skopacz1) [reply updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc:137]: I can rephrase this sentence if it's hard to understand. I think it's explaining how the CVO uses release image signatures to verify that the release images you are trying to apply during an update match the images that Red Hat serves (and thus haven't been modified by someone else).

Maybe the note could be rephrased to this:
```suggestion
The Cluster Version Operator (CVO) uses release image signatures to ensure that release images have not been modified, by verifying that the release image signatures match the expected result.
```

Let me know if that rephrasing would make more sense to you.

File: modules/configure-cvo-retrieve-image-signatures.adoc (hunk 8 above)
Comment (JianLi-RH) [reply modules/configure-cvo-retrieve-image-signatures.adoc:27]: I copy the command line from UI to notebook, then update quotations, then run it.

Copy it to terminal doesn't work.

File: updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc (hunk 11 above)
Comment (JianLi-RH) [reply updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc:137]: Cool! Right now it is very clear for me. Thank you @skopacz1

File: updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc [hunk 13]
```diff
@@ -114,24 +114,27 @@ and must be no more than 63 characters`, try creating the Update Service with a
...
+// Configuring the Cluster Version Operator (CVO) to retrieve release image signatures
+include::modules/configure-cvo-retrieve-image-signatures.adoc[leveloffset=+2]
+
 [id="next-steps_updating-restricted-network-cluster-osus"]
 == Next steps
 
 Before updating your cluster, confirm that the following conditions are met:
 
 * The Cluster Version Operator (CVO) is configured to use your locally-installed OpenShift Update Service application.
-* The release image signature config map for the new release is applied to your cluster.
```
Comment (jiajliu) [inline updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc:130]: original signature config map is still support, so i think we should not remove it, right? so suggest to add `or the cluster is able to retrieve release image signatures for the update target release images from osus`

File: modules/configure-cvo-retrieve-image-signatures.adoc (hunk 8 above)
Comment (skopacz1) [reply modules/configure-cvo-retrieve-image-signatures.adoc:27]: @JianLi-RH understood, thanks for explaining.

@jiajliu I'll remove the `/` from that part of the path, thanks.

File: modules/configure-cvo-retrieve-image-signatures.adoc [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
++
+[source,terminal]
+----
+$ while sleep 1; do SIGNATURES_URI=“$(oc -n “${NAMESPACE}” get -o jsonpath=‘{.status.metadataURI}/api/upgrades_info/signatures/{“\n”}’ updateservice “${NAME}“)”; SCHEME=“${SIGNATURES_URI%%:*}“; if test “${SCHEME}” = http -o “${SCHEME}” = https; then break; fi; done
+----
+
+. Patch the cluster version resource by running the following command:
++
+[source,terminal]
+----
```
[the remark this answers is not in the document]
Comment (skopacz1) [reply modules/configure-cvo-retrieve-image-signatures.adoc:33]: I can add a prerequisite item about enabling that featureset. I think there's a snippet on lines 13 and 14 of this file that state that `signatureStores` is TP, but I don't believe it explicitly states you need to enable `TechPreviewNoUpgrade`.

File: updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc (hunk 13 above)
Comment (skopacz1) [reply updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc:130]: I can keep the old statement while including the new option in the same sentence, thanks

**Comment by skopacz1** [comment]
@JianLi-RH and @jiajliu thank you both for your feedback! Could you please take a look when you have a chance and see if I've addressed everything correctly? Thanks.

File: modules/configure-cvo-retrieve-image-signatures.adoc [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+$ while sleep 1; do SIGNATURES_URI="$(oc -n "${NAMESPACE}" get -o jsonpath='{.status.metadataURI}/api/upgrades_info/signatures{"\n"}' updateservice "${NAME}")"; SCHEME="${SIGNATURES_URI%%:*}"; if test "${SCHEME}" = http -o "${SCHEME}" = https; then break; fi; done
+----
+
+. Patch the cluster version resource by running the following command:
++
+[source,terminal]
+----
+$ oc patch clusterversion version --type=merge -p '{"spec":{"signatureStores":[{"url": "'${SIGNATURES_URI}'"}]}}'
+----
```
Comment (jiajliu) [inline modules/configure-cvo-retrieve-image-signatures.adoc:36]: Because the signaturestore provided by osus is behind self-signed ingress CA, if the ca is not enabled in cluster-wide proxy, CVO provides a config through above api to setting it explicitly. Following is an example steps to add the ca cm.
 
```
# ./oc -n openshift-config create configmap valid-ca --from-file=ca.crt
configmap/valid-ca created

# ./oc patch clusterversion version --type merge -p '{"spec": {"signatureStores": [{"url": "https://update-service-oc-mirror-meta-route-openshift-update-service.apps.jliu416d.qe.gcp.devcluster.openshift.com/api/upgrades_info/signatures","ca": {"name": "valid-ca"}}]}}'
clusterversion.config.openshift.io/version patched
```
I'd suggest to add the note here as a plus configure step.

Comment (jiajliu) [reply modules/configure-cvo-retrieve-image-signatures.adoc:36]: new update about the ca support of signaturestore:
the pr for https://issues.redhat.com/browse/OCPBUGS-24700 has been reverted due to regression hours ago, i guess the ca part which implemented in the bug will miss the scope of 4.16 GA, so maybe we could ignore above comment to mention the set of `ca`. 
 cc @wking @PratikMahajan

Comment (skopacz1) [reply modules/configure-cvo-retrieve-image-signatures.adoc:36]: Is the rest of the content in this section still at risk of not making it to 4.16? I can still proceed with QE review and peer review just in case, but I'm curious if there are any updates on the state of the features this PR is documenting

**Comment by skopacz1** [comment]
Closing this PR as it seems that the signature store feature, which makes up the core of what this PR is documenting, will likely not make it to 4.16.

If it turns out that the feature will still be implemented for 4.16, I'll reopen this PR.
